index.bs

```diff
@@ -1511,6 +1511,28 @@ enum AudioContextLatencyCategory {
 </table>
 </div>
 
+<pre class="idl">
+enum AudioContextSinkCategory {
+		"none"
+};
+</pre>
+	
+<div class="enum-description">
+<table class="simple" dfn-type=enum-value dfn-for="AudioContextSinkCategory">
+	<thead>
+		<tr>
+			<th scope="col" colspan="2">
+				Enumeration description
+	<tbody>
+		<tr>
+			<td>
+				"<dfn>none</dfn>"
+			<td>
+				The audio graph will be rendered by a worker thread without using an
```

Comment:
I don't think we want to be prescriptive here by saying "worker". Let's just say the audio will be processed without being output to an audio output device.